styleguide/source/_patterns/02-organisms/03-sections/07-simple-step-form/simple-step-form.twig

```diff
@@ -0,0 +1,32 @@
+{#
+ # @file simple-step-form.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<form class="simple-step-form" method="get" action="" novalidate="novalidate">
+  <fieldset>
+    <p class="h2">To reach us by email, please fill out the following:</p>
+    {% include 'molecules-input-text' %}
+    {% include 'molecules-input-telephone' %}
+    {% include 'molecules-input-select' %}
+    {% include 'molecules-input-email' %}
+    {% include 'molecules-input-radio' %}
+    {% include 'molecules-input-month' %}
+    {% include 'molecules-input-number' %}
```

Comment:
Is validation input available for year length? I can enter a really distant future.
<img width="665" alt="screen shot 2017-02-27 at 4 29 45 pm" src="https://cloud.githubusercontent.com/assets/4439057/23382992/076d9fe2-fd0a-11e6-812e-9c02db40ac70.png">